composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "webtopay/webtopay-bundle",
     "description": "Symfony2 bundle for WebToPay provided services",
-    "minimum-stability": "dev",
+    "minimum-stability": "stable",
```

Comment:
Does not affect anything except tests if we'd use those here. I'd suggest just removing that